tests/pdctl/store/store_test.go

```diff
@@ -196,6 +196,14 @@ func (s *storeTestSuite) TestStore(c *C) {
 	limit2 = leaderServer.GetRaftCluster().GetStoreLimitByType(2, storelimit.RemovePeer)
 	c.Assert(limit2, Equals, float64(25))
 
+	// store limit all <key> <value> <rate> <type>
+	args = []string{"-u", pdAddr, "store", "limit", "all", "zone", "uk", "20", "remove-peer"}
```

Comment:
can we add a simpler command for TiFlash？ cc @rleungx

Reply:
`store limit all engine tiflash 60` How to reduce some words...

Reply:
maybe we can do it in another PR.